src/app/components/ratingcomp.js

```diff
@@ -0,0 +1,14 @@
+
+// Rectangle.js
+import React from 'react';
+
+const RatingComp = ({ color, text }) => {
+
+    return (
+        <div style={{ backgroundColor: color, width: '300px', height: '100px', display: 'flex', alignItems: 'center', justifyContent: 'center', border: '1px solid #ccc', borderRadius: '8px', marginTop: '20px' }}>
```

Comment:
This could be changed with some tailwind. Using brackets like below will let you add custom sizing @stlbpw 
`className={`w-[300px] h-[100px] flex items-center justify-center border border-gray-300 rounded-lg mt-5`}  style={{ backgroundColor: color }}`

Reply:
Updated